### ISSUE TITLE
\implies LaTex expression doesn't work

### ISSUE THREAD

**Issue by Eugleo** [careteditor/caret#328]
As the title says, if I try to imply something in a math block in Caret, in the preview it just shows "implies" in red. 

Also, while talking about LaTex - how is that auto-rendering supposed to work? On the website it almost looks like it'd render math inline.

**Comment by astoilkov** [comment]
Hi @EugLion,

We use [KaTeX](https://github.com/Khan/KaTeX) to render math which doesn't support `\implies` for now. We will track the [issue](https://github.com/Khan/KaTeX/issues/190) regarding `\implies` and update Caret when KaTeX implement it.

Regarding the auto-rendering: It currenlty renders only block math expressions like the example shown below. Note that the cursor should not be inside of the block or the line next to it.

```
$$$
f(x) = \int_{-\infty}^\infty
    \hat f(\xi)\,e^{2 \pi i \xi x}
    \,d\xi
$$$
```

**Comment by kevindoveton** [comment]
@astoilkov this was added in [0.8.0](https://github.com/Khan/KaTeX/releases)

**Comment by astoilkov** [comment]
Indeed. I also saw this today. We will update Latex soon.

**Comment by WesleyYue** [comment]
Any news on updating katex?

**Comment by astoilkov** [comment]
Yes. We will need between one and two weeks before we release a new version of Caret.

**Comment by astoilkov** [comment]
The update of KaTeX is now available in the new [Caret 4.0 Beta](https://caret.io/4beta/).

Closing this issue. Sorry for the delayed release.
